### DISCUSSION TITLE
API for creating logs?

### DISCUSSION THREAD

**Discussion by jakubtomas108** [pocketbase/pocketbase#1781] [Q&A] [answered]
Hello, just an idea. Is there any plan to create Web API for creating logs? I have several "unrelated" servers, PocketBase in center off it all (since it has nice UI). It would be great if I could create logs from the non-PocketBase server to PocketBase through API. It would mean that all of my logs are accessible from one place. I think it would be good even for FE apps, to log exceptions this way. What do you think?

**Comment by ganigeorgiev** [comment, marked as answer]
Generalized logs (and option to submit logs from 3rd party applications) is in the [roadmap](https://github.com/orgs/pocketbase/projects/2/views/1) and will be implemented sometime in the future but it is pushed back for now due to other more critical/blocker tasks.

**Reply by Partysun** [reply]
Hi. I see in the roadmap "generalized logs" in the done section. Do we have any updates here, maybe not in a mojor version?

**Reply by ganigeorgiev** [reply]
@Partysun The logs are already refactored. See:
- Go: https://pocketbase.io/docs/go-logging/
- JS: https://pocketbase.io/docs/js-logging/

If that's not what you are looking for, please elaborate more in a new Q&A discussion.

**Reply by ganigeorgiev** [reply]
@Partysun And if you were looking for a web API see https://github.com/pocketbase/pocketbase/issues/4800.

**Reply by Partysun** [reply]
> It would be great if I could create logs from the non-PocketBase server to PocketBase through API

@ganigeorgiev  I am looking for a Client Side SDK for logs. 
Maybe, it's already here and I don't see it ;)

**Reply by ganigeorgiev** [reply]
As explained in the above [linked issue](https://github.com/pocketbase/pocketbase/issues/4800) there is no such functionality at the moment and even if there was it will be limited only to admins.

**Reply by Partysun** [reply]
Thank you for quick answer, I appreciate thit vision. Btw, great product.
